package.json

```diff
@@ -9,40 +9,41 @@
     "url": "https://github.com/cardsforhumanity/cardsforhumanity.git"
   },
   "engines": {
-    "node": "0.10.x",
-    "npm": "1.2.x"
+    "node": "6.10.x",
+    "npm": "3.10mongoo.x"
```

Comment:
This is required for deploying the application to Heroku, It specifies the version of npm and node that runs on heroku